sdcm/nemesis.py

```diff
@@ -602,7 +602,7 @@ def disrupt_stop_wait_start_scylla_server(self, sleep_time=300):  # pylint: disa
         time.sleep(sleep_time)
         if self._is_it_on_kubernetes():
             # Kubernetes brings node up automatically, no need to start it up
-            self.target_node.wait_db_up(timeout=sleep_time)
+            self.target_node.wait_node_fully_start(timeout=sleep_time)
```

Comment:
this one i'm not completely sure if this is correct... perhaps @vponomaryov can confirm it

Reply:
From a brief look at a code references looks compatible.
Let it be.